standalone/standalone/src/Runner.ts

```
@@ -139,7 +139,10 @@ export class Runner {
     } else if (options?.innerObjectHandlers) {
       Object.assign(this.innerObjects, options.innerObjectHandlers);
     }
-    this.loadUnitLoader = new EggModuleLoader(this.moduleReferences);
+    this.loadUnitLoader = new EggModuleLoader(this.moduleReferences, {
+      logger: ((this.innerObjects.logger && this.innerObjects.logger[0])?.obj as Logger) || console,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)